src/CommunityToolkit.Aspire.Hosting.EventStore/EventStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,209 @@
+    }
+
+    /// <summary>
+    /// Adds a bind mount for the data folder to a EventStore container resource.
+    /// </summary>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="source">The source directory on the host to mount into the container.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    /// <remarks>
+    /// <example>
+    /// Add an EventStore container to the application model and reference it in a .NET project. Additionally, in this
+    /// example a bind mount is added to the container to allow data to be persisted across container restarts.
+    /// <code lang="csharp">
+    /// var builder = DistributedApplication.CreateBuilder(args);
+    ///
+    /// var eventstore = builder.AddEventStore("eventstore")
+    ///   .WithDataBindMount("./data/eventstore/data");
+    /// var api = builder.AddProject&lt;Projects.Api&gt;("api")
+    ///   .WithReference(eventstore);
+    ///  
+    /// builder.Build().Run(); 
+    /// </code>
+    /// </example>
+    /// </remarks>
+    public static IResourceBuilder<EventStoreResource> WithDataBindMount(this IResourceBuilder<EventStoreResource> builder, string source)
+    {
+        ArgumentNullException.ThrowIfNull(builder);
+        ArgumentNullException.ThrowIfNull(source);
+
+        return builder.WithBindMount(source, DataTargetFolder);
+    }
+
+    /// <summary>
+    /// Adds a named volume for the log folder to a EventStore container resource.
+    /// </summary>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="name">The name of the volume. Defaults to an auto-generated name based on the application and resource names.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    /// <remarks>
+    /// <example>
+    /// Add an EventStore container to the application model and reference it in a .NET project. Additionally, in this
+    /// example a log volume is added to the container to allow logs to be persisted across container restarts.
+    /// <code lang="csharp">
+    /// var builder = DistributedApplication.CreateBuilder(args);
+    ///
+    /// var eventstore = builder.AddEventStore("eventstore")
+    ///   .WithLogVolume();
+    /// var api = builder.AddProject&lt;Projects.Api&gt;("api")
+    ///   .WithReference(eventstore);
+    ///  
+    /// builder.Build().Run(); 
+    /// </code>
+    /// </example>
+    /// </remarks>
+    public static IResourceBuilder<EventStoreResource> WithLogVolume(this IResourceBuilder<EventStoreResource> builder, string? name = null)
+    {
+        ArgumentNullException.ThrowIfNull(builder);
+
+#pragma warning disable CTASPIRE001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+        return builder.WithVolume(name ?? VolumeNameGenerator.CreateVolumeName(builder, "log"), LogTargetFolder);
```

Comment:
Why we have seperate method for this instead of using this in `WithDataVolume`?

Reply:
This one is for mounting logs (`/var/logs/eventstore`) while the other one mounts data (`/var/lib/eventstore`).
Should we combine both? I usually don't store logs as it can get pretty big.

Reply:
If users usually don't store logs that should be fine.